### PR TITLE
fix(ui): add internal notification when copying payment request

### PR DESCRIPTION
**Pull request by korhaliv**
## Description:
Add internal (inside GUI) notification when copying payment request

## Motivation and Context:
Extend existing error notification system to also  handle warning and success notifications. This allows to give user a feedback on certain actions where this is expected. This enhancement was utilized to address #1380 

## How Has This Been Tested?
Manually. Created a payment request and pressed copy invoice

## Screenshots (if appropriate):
image

## Types of changes:
New feature

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have reviewed and updated the documentation accordingly.
- [x] I have read the _CONTRIBUTING_ document.
- [ ] I have added tests to cover my changes where needed.
- [x] All new and existing tests passed.
- [x] My commits have been squashed into a concise set of changes.

### CODE REVIEW

**Comment by mrfelton** [comment]
I would suggest that you add that delay method as part of this PR, and then when we forward merge this fix into next it will just cancel out with what is already in that branch.

**Comment by mrfelton** [comment]
It seems there are still some random test failures when the e2e tests run in the CI environment. It's not specifically related to this PR. Re-running the failed build usually gets the build to go green.

File: app/reducers/notification.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    type: REMOVE_NOTIFICATION,
+    id
+  }
+}
+
+export const receiveError = (event, error) => dispatch => {
+  dispatch(setError(error))
+}
+
+export const setError = (message, delay = NOTIFICATION_TIMEOUT) =>
```
[the remark this answers is not in the document]
Comment (korhaliv) [reply app/reducers/notification.js]: done

File: app/containers/Home.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ const mapDispatchToProps = {
   unlockWallet,
   deleteWallet,
   setIsWalletOpen,
-  setError
+  setError: showError
```
Comment (mrfelton) [inline app/containers/Home.js]: Why retain the old prop name here?

**Comment by korhaliv** [comment]
Updated

**Pull request merged**